ajax/libs/zrender/2.1.1/zrender.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2006 Google Inc.
```

Comment:
This file contain nothing but comments. Do we need to include this file?

Reply:
P.S. `zrender.js` seems empty from 1.0.3 ~ 2.1.1

Reply:
Please take a look line 29
This file can work by these codes.
https://github.com/ecomfe/zrender/blob/2.1.1/build/zrender.js#L29

Reply:
@hare1039 Please take a look line 29
This file can work by these codes.
https://github.com/ecomfe/zrender/blob/2.1.1/build/zrender.js#L29